src/stash/src/postgres.rs

```diff
@@ -260,6 +263,27 @@ impl Postgres {
         Ok(())
     }
 
+    /// Verifies stash invariants. Should only be called by tests.
+    pub async fn verify(&self) -> Result<(), StashError> {
```

Comment:
Tried, and apparently this isn't possible. I'd guess that because the stash tests are integration tests, the `test` cfg isn't enabled for the main crate? Dunno.